src/type/validate.js

```diff
@@ -355,6 +358,21 @@ function validateObjectInterfaces(
   });
 }
 
+function validateInterfaces(
+  context: SchemaValidationContext,
+  iface: GraphQLInterfaceType,
+): void {
+  const possibleTypes = context.schema.getPossibleTypes(iface);
+
+  if (possibleTypes.length === 0) {
+    context.reportError(
+      `Interface ${iface.name} must be implemented` +
```

Comment:
Could you please expand the tests further to include a test which asserts this error occurs?